### PR TITLE
Rewrite Numbers overview and guides

### PR DESCRIPTION
**Pull request by marklewin**
## Description

Rewrite the overview page to stop it just being a rehash of what's available in the guides. Add new guides.

### CODE REVIEW

File: _documentation/numbers/guides/numbers-cli.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+Read the [installation instructions](https://github.com/Nexmo/nexmo-cli#installation) to get started.
+
+## List your numbers
+
+The `nexmo numbers:list` command lists all the numbers owned by the  account.
+
+Optional flags are:
+
+* `--size`: The number of results to return
```
Comment (tbedford) [inline _documentation/numbers/guides/numbers-cli.md]: Neater in a table?

File: _documentation/numbers/guides/payments.md [hunk 2]
```diff
@@ -1,29 +1,24 @@
...
+One-click payments are quick and easy transactions made with a credit card stored for your account. You don't need to re-enter your credit card details as these are stored with our Payment Service Provider (PSP), [Braintree](https://www.braintreepayments.com/). Braintree (owned by PayPal) is [PCI level 1 compliant](https://www.braintreepayments.com/gb/features/data-security).
 
-One-click payments are quick and easy transactions made with a credit card stored for your account. You are not required to re-enter your credit card details as these are stored with our Payment Service Provider (PSP), Braintree. Braintree (owned by PayPal) is PCI level 1 compliant. We do not store, manage or transmit any complete credit card information as part of our PCI Compliance certification. Nexmo has submitted and passed PCI DSS Self-Assessment Questionnaire Merchant Compliance. We can produce evidence of this upon request.
+> We do not store, manage or transmit any complete credit card information as part of our PCI Compliance certification. Nexmo has submitted and passed PCI DSS Self-Assessment Questionnaire Merchant Compliance. We can produce evidence of this upon request.
```
Comment (tbedford) [inline _documentation/numbers/guides/payments.md]: Nexmo instead of 'we'?

File: _documentation/numbers/guides/payments.md [hunk 3]
```diff
@@ -32,77 +27,87 @@ You can use Dashboard to
...
-1. On the top-right of Dashboard, click the arrow next to **&lt;username>**, then click **Billing & payments**.
-2. Select an amount, click **Add a new payment method**, then click **Make payment**.
-3. Choose a **Payment method**, fill in the **Billing method** and click **Next**.
-  * PayPal - you are directed to the PayPal website to complete payment.
-  * Credit card - enter your card details. Your payment is processed in the Dashboard.
-  **Note**: You must provide the address associated with your credit card or PayPal account to successfully complete payment.
+1. Sign in to the [developer dashboard](https://dashboard.nexmo.com).
+2. In the left-hand navigation menu, click the arrow next to your user name, then click **Billing & payments**.
+3. If you see an "Upgrade your account" message, click **Upgrade account**.
+3. Select a payment method, payment amount and complete your billing address details and then click **Next**. Note that you must use the address associated with the card or Paypal account for the payment to be successful.
```
Comment (tbedford) [inline _documentation/numbers/guides/payments.md]: Duplicated 3.

File: _documentation/numbers/guides/numbers-cli.md (hunk 1 above)
Comment (marklewin) [reply _documentation/numbers/guides/numbers-cli.md]: Much neater!

File: _documentation/numbers/guides/payments.md (hunk 2 above)
Comment (marklewin) [reply _documentation/numbers/guides/payments.md]: Changed

File: _documentation/numbers/guides/payments.md (hunk 3 above)
Comment (marklewin) [reply _documentation/numbers/guides/payments.md]: Fixed

**Comment by marklewin** [comment]
Thanks for the helpful suggestions Tony, all of which I have implemented and would appreciate a second review.

File: _documentation/numbers/guides/number-management.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+## Configure a virtual number
+
+To configure a Nexmo virtual number:
+
+1. Sign in to the [developer dashboard](https://dashboard.nexmo.com).
+2. In the left-hand navigation menu, click **Numbers** then **Your numbers**.
+3. Click the gear icon next to the number you want to configure.
+4. Edit your configuration as required and then click **OK**. (If you are changing a [webhook endpoint](/concepts/guides/webhooks), ensure that the new webhook endpoint is live first.)
+
+## Enable two-factor authentication
```
Comment (mheap) [inline _documentation/numbers/guides/number-management.md]: This feels like a separate guide to me. It is done in the dashboard, but we don't have a corresponding entry in the CLI guide

File: _documentation/numbers/guides/number-management.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+When you use a pre-approved US short code from Nexmo, you **MUST** display the following information on your site's opt-in page:
+
+  * Frequency: How often users of your service can expect to receive messages from you
+  * How to opt out: Send a `STOP` SMS to your short code.
+  * How to get help: Send a `HELP` SMS to your short code.
+  * The cost to your users to receive your messages (message and data rate)
+  * The terms and conditions for your service
+  * Your privacy policy
+
+## Setup event alerts
```
Comment (mheap) [inline _documentation/numbers/guides/number-management.md]: As above, this feels like a separate guide to me

File: _documentation/numbers/guides/numbers-cli.md [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+
+Optional flags are:
+
+| Flag | Description |
+|---|---|
+| `--size`  | The number of results to return  |
+| `--page` | The number of results per page  |
+| `--pattern` | The pattern you want to search for. Use the `*` wildcard to match the start or end of the number. For example, `*123*` matches all numbers that contain the pattern `123`.  |
+
+Aliases: `nexmo nl`, `nexmo numbers` and `nexmo number:list`.
```
Comment (mheap) [inline _documentation/numbers/guides/numbers-cli.md]: If there are multiple ways to do something, let's choose our favourite (`numbers:list` in this case) and only show that way

File: _documentation/numbers/guides/number-management.md (hunk 4 above)
Comment (marklewin) [reply _documentation/numbers/guides/number-management.md]: I think you're absolutely right. I've moved the 2fa and event alert information to separate guides.

File: _documentation/numbers/guides/number-management.md (hunk 5 above)
Comment (marklewin) [reply _documentation/numbers/guides/number-management.md]: As above, I agree.

File: _documentation/numbers/guides/numbers-cli.md (hunk 6 above)
Comment (marklewin) [reply _documentation/numbers/guides/numbers-cli.md]: Removed the aliases.

**Pull request merged**